doc/user/layouts/shortcodes/self-managed/terraform-disclaimer.html

```diff
@@ -0,0 +1,9 @@
+The sample Terraform module is for **evaluation purposes only** and not intended
+for production use. It is provided to help you get started with Materialize for
+evaluation purposes only. Materialize does not support nor recommends this
+module for production use. Materialize does not guarantee tests for changes to
```

Comment:
I do love the warning here, but the first three lines are all a little redundant... might be able to pare back

Reply:
Updated in patch